src/mouse.js

```diff
@@ -4,7 +4,8 @@ var pointermap = dispatcher.pointermap;
 // radius around touchend that swallows mouse events
 var DEDUP_DIST = 25;
 
-var WHICH_TO_BUTTONS = [0, 1, 4, 2];
+// left, middle, right, back, forward
+var BUTTON_TO_BUTTONS = [1, 4, 2, 8, 16];
```

Comment:
> I don’t have one to test but I know there are 10-button mice on the market.

The WarMouse Meta actually has [18 buttons](http://www.engadget.com/2010/05/03/warmouse-meta-review/). That said, the only buttons I've seen supported by operating systems / UAs beyond the left, right, and middle buttons are the two thumb buttons, popularly introduced by the [IntelliMouse](https://en.wikipedia.org/wiki/IntelliMouse).